bigquery/bigquery.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: example-bigquery-notifier
 spec:
   notification:
-    filter: build.build_trigger_id == "123e4567-e89b-12d3-a456-426614174000" 
+    filter: build.build_trigger_id == "example_id"
```

Comment:
Up to you: it might be good to leave this as the "fake" UUID since it shows customers that they can't use their trigger names here.